embuilder.py

```diff
@@ -33,7 +33,6 @@
         libcxx_noexcept
         libcxxabi
         gl
-        struct_info
```

Comment:
the test is in `test_sanity.py` for that reason - those tests can break the install (but shouldn't ;)